integration_tests/internal/pkgr.yml

```diff
@@ -22,14 +22,13 @@ Suggests: true
 Repos:
   - r_validated: "https://metrumresearchgroup.github.io/r_validated"
   - CRAN: "https://cran.rstudio.com"
-  - local: "path/on/filesystem"
```

Comment:
if we're going to test local, it needs to be a relative path to the pkgr file so it can also be generalized across machines

Reply:
Right, I haven't finished cleaning this up but I figured I'd move the ones I messed with into their own folders for now.